### DISCUSSION TITLE
Toast message dismiss animation broken

### DISCUSSION THREAD

**Discussion by Ageiris** [quasarframework/quasar#14843] [General - Components / Directives / etc] [answered]
When a toast message gets dismissed, for some unknown reason, the transition breaks and starts from the right edge of the screen no matter where the toast message is displayed (center, top, bottom, middle).

![transition glitch](https://user-images.githubusercontent.com/10961431/200367783-0179a872-ea38-4d35-a858-5ba96f6735b6.gif)

Are we doing something wrong? Even if we use the simplest toast available, the transition breaks. 
Using Quasar 2.10.1

**Comment by pdanpdan** [comment]
Do you have a reproduction for that?
What browser/os?

**Reply by Ageiris** [reply]
We can't reproduce it on Quasar Documentation pages. There everything works. 
It is broken when we use it within our VueJS 3 app. Tested on Windows 11 & Edge / Windows 10 & Firefox

**Comment by pdanpdan** [comment, marked as answer]
Do you have any fixed/absolute positioning on html/body/the div that holds the notifications?
Make a persistent notification and then inspect the styles of it's parents

**Reply by Ageiris** [reply]
Thanks @pdanpdan . Your suggestion inspired us to look into our global CSS files and we really had some weird positions there. When we removed them, it started to work again.
